client/randmovieapp/src/test.js

```diff
@@ -0,0 +1,17 @@
+function addition(x, y) {
+    let a = x + 1
+    let b = x + 1
+    let c = a + b
+    return c
+  }
+
+  function addition2() {
+    let a = x + 1
+    let b = x + 1
```

Comment:
Codacy found an issue: ['x' is not defined.](https://app.codacy.com/gh/Reyvenk/Randmovie/pullRequest?prid=6284384)